### PR TITLE
feat: keyColumnName / fix: IE9 issues

### PR DESCRIPTION
**Pull request by dongwoo-kim**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
- Add `keyColumnName` options
- Fix IE9 issues 
  - `requestAnimationFrame` -> `setTimeout`
  - remove `Array.prototype.findIndex`

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/helper/common.ts [hunk 1]
```diff
@@ -99,6 +85,20 @@ export function findPropIndex<T>(propName: keyof T, value: T[keyof T], arr: T[])
   return findIndex((item) => item[propName] === value, arr);
 }
 
+export function findIndexes<T>(predicate: (v: T) => boolean, arr: T[]) {
+  return arr.reduce((acc, v, idx) => (predicate(v) ? [...acc, idx] : acc), [] as number[]);
+}
+
+export function findPrevIndex<T>(arr: T[], predicate: (_: T) => boolean): number {
+  const index = findIndex(predicate, arr);
```
Comment (dongwoo-kim) [inline src/helper/common.ts]: 이 라인만 수정되었습니다. 나머지는 자리 이동이에요 ㅎ

File: src/store/columns.ts [hunk 2]
```diff
@@ -145,13 +154,18 @@ export function create(
...
     frozenCount: columnOptions.frozenCount || 0,
     allColumns,
 
     get allColumnMap() {
       return createMapFromArray(this.allColumns, 'name') as Dictionary<ColumnInfo>;
     },
 
+    get rowHeaderCount() {
+      return rowHeaderInfos.length;
+    },
```
Comment (dongwoo-kim) [inline src/store/columns.ts]: 이제 computed 값들이 순서에 영향을 받습니다.

Comment (seonim-ryu) [reply src/store/columns.ts]: 순서에 영향을 받는다는게.. 선언된 순서를 말하는건가요??

**Comment by seonim-ryu** [comment]
[5/31] 리뷰 완료! 저도 이상 없습니다ㅎ 트리에도 keyColumnName을 처리해야겠군요..

File: src/store/columns.ts (hunk 2 above)
Comment (dongwoo-kim) [reply src/store/columns.ts]: 네. computed 함수 내에서 사용하는 값들이 먼저 선언되어야 합니다.

**Comment by js87zz** [comment]
저도 리뷰완료입니다!고생하셨습니다

**Pull request merged**